test/celeritas/em/SeltzerBerger.test.cc

```diff
@@ -262,39 +279,23 @@ TEST_F(SeltzerBergerTest, sb_energy_dist)
             // Loop over many particles
             sample_many(inc_energy, sample_energy);
         }
-
-        // Sample with the positron XS correction
-        {
-            ParticleParams const& pp = *this->particle_params();
-
-            SBEnergyDistribution<SBPositronXsCorrector> sample_energy(
-                edist_helper,
-                {pp.get(pp.find(pdg::positron())).mass(),
-                 this->material_params()->get(ElementId{0}),
-                 gamma_cutoff,
-                 Energy{inc_energy}});
-
-            // Loop over many particles
-            sample_many(inc_energy, sample_energy);
-        }
     }
 
     // clang-format off
-    const double expected_max_xs[] = {2.866525852195, 4.72696244794,
-        12.18911946078, 13.93366489719, 13.85758694967, 13.3353235437};
-    const double expected_xs_zero[] = {1.98829818915769, 4.40320232447369,
+    static double const expected_max_xs[] = {2.866525852195, 4.72696244794,
         12.18911946078, 13.93366489719, 13.85758694967, 13.3353235437};
     static double const expected_avg_exit_frac[] = {0.94912259860422,
-        0.49765332179155, 0.082254084865518, 0.064928479979342,
-        0.077435723446377, 0.089209770419741, 0.064206311258969,
-        0.064814238042425};
+        0.90270157074556, 0.49736065674058, 0.27711716215819,
+        0.081515129333292, 0.068559142299853, 0.065803331441324,
+        0.064344514250384, 0.079512002547402, 0.077647502218254,
+        0.085615341879476, 0.086428313853775, 0.065321200129584};
     static double const expected_avg_engine_samples[] = {4.0791015625,
-        4.060546875, 5.13623046875, 4.6572265625, 4.43115234375, 4.35791015625,
-        9.3740234375, 4.65478515625};
+        137.044921875, 4.060546875, 15.74169921875, 5.103515625, 5.26953125,
```

Comment:
That's a lot of rejection, especially considering the comment above:

> Note: the first point has a very low cross section compared to ionization so won't be encountered in practice. The differential cross section distribution is much flatter there, so there should be lower rejection. 

Do you have any plots of the maximum cross section versus the actual distribution?

Reply:
Good point, I'll update the comment since it only applies to the electron DCS -- for low incident energies the positron DCS deviates a lot. These are cross sections for a 1 keV incident particle (same as that first point in the test) and a few different cutoff energies (0.9 keV is used in the test):
![sb_dcs](https://github.com/celeritas-project/celeritas/assets/6426426/e23f2265-be20-4036-b114-920d38e0aca6)

So, the sampling efficiency for the positrons here will be quite poor. For higher incident energies though the positron DCS will be much closer to the electron DCS.

Reply:
Yuck, that rejection rate is high indeed, but it should be a very unusual case indeed. Thanks for the plot!